server/user_account/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
+
+class CustomUser(AbstractBaseUser):
+    USERS = (
+        ('SA', 'System Admin'),
+        ('IM', 'Inventory Manager'),
+        ('SK', 'Stock Keeper'),
+    )
+    user_name = models.CharField(max_length=30, unique=True)
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    role = models.CharField(max_length=2, choices=USERS)
+    is_active = models.BooleanField(default=True)
+    email = models.EmailField(verbose_name='email', max_length=255, unique=True)
```

Comment:
Why is this a max length of 255, but the names are 30?

Reply:
Because emails are usually longer and its a combination of first name + last name + whatever, but again we can agree on another number.